packages/components/src/elevation/stories/index.tsx

```diff
@@ -0,0 +1,55 @@
+/**
+ * External dependencies
+ */
+import type { ComponentMeta, ComponentStory } from '@storybook/react';
+
+/**
+ * Internal dependencies
+ */
+import { Elevation } from '..';
+
+const meta: ComponentMeta< typeof Elevation > = {
+	component: Elevation,
+	title: 'Components (Experimental)/Elevation',
+	argTypes: {
+		as: { control: { type: 'text' } },
+		borderRadius: { control: { type: 'text' } },
+	},
+	parameters: {
+		controls: {
+			expanded: true,
+		},
+		docs: { source: { state: 'open' } },
+	},
+};
+export default meta;
+
+const Template: ComponentStory< typeof Elevation > = ( args ) => {
+	/* eslint-disable jsx-a11y/anchor-is-valid */
+	return (
+		<a
+			href="#"
+			onClick={ ( e ) => e.preventDefault() }
+			style={ {
+				border: 0,
+				background: 'transparent',
+				display: 'block',
+				width: 150,
+				height: 150,
+				position: 'relative',
+				margin: '20vh auto',
+			} }
+		>
+			<Elevation { ...args } />
+		</a>
+	);
+	/* eslint-enable jsx-a11y/anchor-is-valid */
```

Comment:
We can fix the `jsx-a11y` problem by using a `button` instead:

```suggestion
	return (
		<button
			style={ {
				border: 0,
				background: 'transparent',
				width: 150,
				height: 150,
				position: 'relative',
			} }
		>
			Click me
			<Elevation { ...args } />
		</button>
	);
```

(Also removed some unnecessary styles for brevity in the code snippets, and added a "Click me" label.)

Reply:
Yes. Smart 👍